Makefile

```diff
@@ -62,6 +62,7 @@ _update_yamls: _set_registry_url
 	sed -i.bak -e 's|che.webhooks.enabled: .*|che.webhooks.enabled: "$(WEBHOOK_ENABLED)"|g' ./deploy/controller_config.yaml
 	sed -i.bak -e 's|che.workspace.default_routing_class: .*|che.workspace.default_routing_class: "$(DEFAULT_ROUTING)"|g' ./deploy/controller_config.yaml
 	sed -i.bak -e 's|cluster.routing_suffix: .*|cluster.routing_suffix: $(ROUTING_SUFFIX)|g' ./deploy/controller_config.yaml
+	sed -i.bak -e 's|che.workspace.sidecar.image_pull_policy: .*|che.workspace.sidecar.image_pull_policy: $(PULL_POLICY)|g' ./deploy/controller_config.yaml
```

Comment:
Just TODO item for me

it does not seem to work yet
![Screenshot_20200529_180525](https://user-images.githubusercontent.com/5887312/83274899-10cafb00-a1d7-11ea-96e8-9da17c94a879.png)
I'll fix it.